doc/script_commands.txt

```diff
@@ -2260,10 +2260,35 @@ array, shifting all the elements beyond this towards the beginning.
 
 ---------------------------------------
 
-*countinarray <array name>{[<start index>]},<value1>{,<value2>...<valueN>};
+*inarray <array name>,<value>;
+
+This command will return the index of the value inside the array.
+It will return -1 if the value not found.
+
+	setarray @array[0], 100, 200, 300, 400, 500, 600;
+	// So we have made @array[]
+	
+	inarray(@array, 200);
+	//return 1 because the number 200 is the index 1 for the array
+	//another way to say it that @array[1] == 200
+	
+	.@index = inarray(@array, 600);
+	//.@index is now 5 because @array[5] == 600
+	
+	inarray(@array, 100);
+	//return 0 because @array[0] == 100
+	
+	inarray(@array, 800);
+	//return -1 because 800 is not an element of the array @array
+
+For more details, see the sample in 'doc/sample/inarray.txt'.
+
+---------------------------------------
+
+*countinarray <array name>,<value1>{,<value2>...<valueN>};
 
 This command will check for matches between the array values and return the number of matches.
-While being optional, if [<start index>] is supplied, the search will begin from the given index value.
+While being optional.
```

Comment:
Was this on purpose? You can check out `cleararray` which supports the index value to add to your script command.